Webpack/webpack.config.js

```diff
@@ -0,0 +1,45 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    path: path.resolve(__dirname, "build"),
+  },
+  plugins: [
+    new BundleAnalyzerPlugin({
+        analyzerMode: 'static',
+        openAnalyzer: true,
+        analyzerPort: 3002,
+      }),
+    new HtmlWebpackPlugin({
+      template: path.resolve(__dirname, "src", "index.html"),
+    }),
+  ],
+  module: {
+    rules: [
+      {
+        test: [/\.css$/, /\.scss$/],
+        use: ["style-loader", "css-loader", "sass-loader"],
+      },
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: ["babel-loader"],
+      },
+    ],
+  },
+  optimization: {
```

Comment:
Add some other optimizations to reduce the build size, please

https://webpack.js.org/plugins/

https://blog.bitsrc.io/5-methods-to-reduce-javascript-bundle-size-67f2e1220457#:~:text=Using%20Production%20Mode%20in%20Webpack,and%20newlines%20in%20the%20code.